OneSchedule.DomainModels/DomainEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OneSchedule.DomainModels.Abstraction;
+using System;
+using System.Collections.Generic;
+
+namespace OneSchedule.DomainModels
+{
+    public class DomainEvent : DomainModel
```

Comment:
Related to the general comments at the top of PR, you can name abstract classes like Base*suffix*